PacketDotNet/WakeOnLanPacket.cs

```diff
@@ -50,6 +51,9 @@ public sealed class WakeOnLanPacket : Packet
 #pragma warning restore 0169, 0649
 #endif
 
+        // the position of password if it is available.
+        private const int PasswordLocation = 102;
```

Comment:
Location -> Position. This should be moved to WakeOnLanFields.

Reply:
Sure, will make the changes as mentioned.